plugin/parser/differ/mysql/differ.go

```diff
@@ -159,6 +180,27 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 	return deparse(newNodeList, inplaceUpdate, dropNodeList, format.DefaultRestoreFlags|format.RestoreStringWithoutCharset)
 }
 
+// We're not support all the MySQL statements for now.
```

Comment:
```suggestion
// We're not supporting all the MySQL statements for now.
```

Also document a list of things that are not supported.

Reply:
Yeah, I will keep improving it in the following PRs.